bitrise.yml

```diff
@@ -244,8 +244,8 @@ workflows:
       - xcode-test-without-building:
           title: Unit Tests
           inputs:
-            - xctestrun: '${BITRISE_TEST_BUNDLE_PATH}/Pocket (iOS)_UnitTests_iphonesimulator16.0.xctestrun'
-            - destination: platform=iOS Simulator,name=iPhone 14,OS=16
+            - xctestrun: '${BITRISE_TEST_BUNDLE_PATH}/Pocket (iOS)_UnitTests_iphonesimulator16.1.xctestrun'
+            - destination: platform=iOS Simulator,name=iPhone 14,OS=16.1
```

Comment:
assuming we will want to be able to handle multiple OS, devices in the future right? In the meantime, are we trying to be consistent with the latest versions?

Reply:
@cyndichin yea. Apple did apple things, and it looks like the commandline tool to install multiple sims on different os disappeared... Still trying to figure out how to do that.